src/abi/connectors/v2/QI-A.ts

```diff
@@ -0,0 +1,3 @@
\n             \"type\": \"bytes\",\n             \"name\": \"_eventParam\",\n             \"internalType\": \"bytes\"\n          }\n       ],\n       \"name\": \"ClaimReward\",\n       \"inputs\": [\n          {\n             \"type\": \"uint256\",\n             \"name\": \"setId\",\n             \"internalType\": \"uint256\"\n          }\n       ]\n    },\n    {\n       \"type\": \"function\",\n       \"stateMutability\": \"payable\",\n       \"outputs\": [\n          {\n             \"type\": \"string\",\n             \"name\": \"_eventName\",\n             \"internalType\": \"string\"\n          },\n          {\n             \"type\": \"bytes\",\n             \"name\": \"_eventParam\",\n             \"internalType\": \"bytes\"\n          }\n       ],\n       \"name\": \"ClaimRewardThree\",\n       \"inputs\": [\n          {\n             \"type\": \"string[]\",\n             \"name\": \"supplyTokenIds\",\n             \"internalType\": \"string[]\"\n          },\n          {\n             \"type\": \"string[]\",\n             \"name\": \"borrowTokenIds\",\n             \"internalType\": \"string[]\"\n          },\n          {\n             \"type\": \"uint256\",\n             \"name\": \"setId\",\n             \"internalType\": \"uint256\"\n          }\n       ]\n    },\n    {\n       \"type\": \"function\",\n       \"stateMutability\": \"payable\",\n       \"outputs\": [\n          {\n             \"type\": \"string\",\n             \"name\": \"_eventName\",\n             \"internalType\": \"string\"\n          },\n          {\n             \"type\": \"bytes\",\n             \"name\": \"_eventParam\",\n             \"internalType\": \"bytes\"\n          }\n       ],\n       \"name\": \"ClaimRewardTwo\",\n       \"inputs\": [\n          {\n             \"type\": \"string[]\",\n             \"name\": \"tokenIds\",\n             \"internalType\": \"string[]\"\n          },\n          {\n             \"type\": \"uint256\",\n             \"name\": \"setId\",\n             \"internalType\": \"uint256\"\n          }\n       ]\n    },\n    {\n       \"type\": \"function\",\n       \"stateMutability\": \"payable\",\n       \"outputs\": [\n          {\n             \"type\": \"string\",\n             \"name\": \"_eventName\",\n             \"internalType\": \"string\"\n          },\n          {\n             \"type\": \"bytes\",\n             \"name\": \"_eventParam\",\n             \"internalType\": \"bytes\"\n          }\n       ],\n       \"name\": \"delegate\",\n       \"inputs\": [\n          {\n             \"type\": \"address\",\n             \"name\": \"delegatee\",\n             \"internalType\": \"address\"\n          }\n       ]\n    },\n    {\n       \"type\": \"function\",\n       \"stateMutability\": \"view\",\n       \"outputs\": [\n          {\n             \"type\": \"string\",\n             \"name\": \"\",\n             \"internalType\": \"string\"\n          }\n       ],\n       \"name\": \"name\",\n       \"inputs\": []\n    }\n ]"
```

Comment:
@Vaibhav20201 ABI doesn't look it's in json format? It's in string format

Reply:
@thrilok209 Solved!